### PR TITLE
Fix Coding Standard Violations

### LINKED ISSUE #197: Code Review: @FeliciaBeatrice
@FeliciaBeatrice 

**src/main/java/seedu/duke/parsermanager/ParseCheckClient.java** :
image
image
image
image

1. Line 33,34: These variable names should be in plural form as it refers to a collection of objects. Currently it also sounds like a function which can be a bit misleading.
2. Line 59,60: This variable name also sounds like a function but is used to represent a collection - avoid misleading names.
3. Line 60: Avoid magic numbers, maybe can use FIRST_INDEX constant to explain the meaning more clearly.
4. Line 129,283,288: Avoid magic strings, maybe can use a constaint to represent string indent since it is repeated.
5. Line 85: Avoid magic numbers

**src/main/java/seedu/duke/command/CommandDeleteProperty.java**

image
1. Line 29: Comment minimally but sufficiently: try not to include comments that explain self explanatory code

### PR DESCRIPTION
**Pull request by FeliciaBeatrice**
Fixes #197

### CODE REVIEW

File: src/main/java/seedu/duke/Ui.java [hunk 1]
```diff
@@ -139,11 +143,11 @@ public void showPairedPropertiesDeletedConfirmationMessage(Client deletedClient,
 
     public void showPairedClientsDeletedConfirmationMessage(Property deletedProperty, PairingList pairingList) {
         HashMap<Client, Property> clientPropertyPairs = pairingList.getClientPropertyPairs();
-        int currentIndex = 1;
+        int currentIndex = FIRST_INDEX;
 
         for (Map.Entry<Client, Property> entry : clientPropertyPairs.entrySet()) {
             if (entry.getValue().equals(deletedProperty)) {
-                if (currentIndex == 1) {
+                if (currentIndex == FIRST_INDEX) {
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Ui.java:150]: Good job converting magic literals to constants

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/duke/Ui.java [hunk 2]
```diff
@@ -55,8 +55,12 @@ public class Ui {
     private static final String CHECK_PROPERTY_FORMAT = "  %d. %s";
 
     private static final String SPACE = "\t";
+    private static final String NEXT_LINE = "\n";
+    private static final String DOUBLE_SPACE = "  ";
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Ui.java:59]: Good job in converting the magic literals to a string with meaningful name.

File: src/main/java/seedu/duke/parsermanager/ParseCheckClient.java [hunk 3]
```diff
@@ -72,8 +73,8 @@ private void checkForExtraFlags(int[] flagIndexPositions) throws ExtraFlagsExcep
         }
     }
 
-    private void validateCheckClientDetails(ArrayList<Integer> checkClientDetails) throws InvalidIndexException {
-        int clientIndex = checkClientDetails.get(0);
+    private void validateCheckClientDetails(ArrayList<Integer> clientDetails) throws InvalidIndexException {
+        int clientIndex = clientDetails.get(0);
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/parsermanager/ParseCheckClient.java:77]: Can consider replacing '0' with a more meaningful name

**Review by wilsonngja — APPROVED** [review]
Overall LGTM. Great job

**Pull request merged**